tests/observability/metrics/conftest.py

```diff
@@ -1149,6 +1179,103 @@ def migration_succeeded_scope_class(vm_migration_metrics_vmim_scope_class):
     )
 
 
+@pytest.fixture()
+def stopped_vm_metric_1(vm_metric_1):
+    vm_metric_1.stop(wait=True)
+
+
+@pytest.fixture()
+def stopped_windows_vm(windows_vm_for_test):
+    windows_vm_for_test.stop()
+    yield windows_vm_for_test
+    running_vm(vm=windows_vm_for_test)
+
+
+@pytest.fixture(scope="module")
+def pvc_with_non_existent_pv(namespace, admin_client):
```

Comment:
same applies here - one test (regardless of os) can be written for this use case